### ISSUE TITLE
Create POST and GET routes for the API

### ISSUE THREAD

**Issue by JosetheJedi** [rawrool/Group7_SmartGarden#42]
I will make the basic POST, and GET routes for the API. Making sure that when people make calls to the API, they get the correct response.

**Comment by JosetheJedi** [comment]
ran into some problems with the server application. I was running a newer version of NodeJS so the new syntax that was being used in one of the dependencies was making the application crash. 

It took an hour to find the bug. I updated all of the dependencies as well to get rid of the deprecated functions that were giving me the warnings. It took about half an hour to research and apply the changes successfully.

**Comment by JosetheJedi** [comment]
got a bit distracted with redoing the database for the server. I was watching more tutorials on building the API and found an easier way to query data without making repeated calls to the server. 

I spent 4 hours on this. I got the main profile page up to date with the new database. I also got the current API routes up to date with the new database.

This new database implementation will make it easier to make the routes for the Plants and the Logs.
